### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by LeonXu98**
Add missing translation.

### CODE REVIEW

File: src/main/resources/assets/modmenu/lang/zh_cn.json [hunk 1]
```diff
@@ -2,12 +2,20 @@
...
   "modmenu.issues": "反馈",
   "modmenu.authorPrefix": "作者: %s",
-  "modmenu.search": "搜索mods"
+  "modmenu.search": "搜索模组",
+  "modmenu.clientsideOnly": "客户端",
+  "modmenu.library": "运行库",
+  "modmenu.toggleFilterOptions": "切换过滤选项",
+  "modmenu.showLibraries": "运行库: %s",
+  "modmenu.showLibraries.true": "显示",
+  "modmenu.showLibraries.false": "隐藏",
```
Comment (liach) [inline src/main/resources/assets/modmenu/lang/zh_cn.json]: `显示中` `已隐藏` sounds better to me.

**Comment by Prospector** [comment]
Thanks for updating these!

File: src/main/resources/assets/modmenu/lang/zh_cn.json [hunk 2]
```diff
@@ -2,12 +2,20 @@
...
   "modmenu.configure": "配置...",
-  "modmenu.modIdToolTip": "Mod ID: %s",
+  "modmenu.modIdToolTip": "模组 ID: %s",
   "modmenu.website": "网站",
   "modmenu.issues": "反馈",
   "modmenu.authorPrefix": "作者: %s",
-  "modmenu.search": "搜索mods"
+  "modmenu.search": "搜索模组",
+  "modmenu.clientsideOnly": "客户端",
+  "modmenu.library": "运行库",
```
Comment (liach) [inline src/main/resources/assets/modmenu/lang/zh_cn.json]: `前置模组` sounds better to me. See https://www.mcbbs.net/thread-908759-1-1.html where `前置/核心Mod` is used.

File: src/main/resources/assets/modmenu/lang/zh_cn.json [hunk 3]
```diff
@@ -2,12 +2,20 @@
   "modmenu.title": "模组",
   "modmenu.loaded": "(已载入%s个)",
   "modmenu.config": "编辑配置",
-  "modmenu.modsFolder": "开启模组资料夹",
-  "modmenu.configsFolder": "开启配置资料夹",
+  "modmenu.modsFolder": "开启模组文件夹",
+  "modmenu.configsFolder": "开启配置文件夹",
   "modmenu.configure": "配置...",
-  "modmenu.modIdToolTip": "Mod ID: %s",
+  "modmenu.modIdToolTip": "模组 ID: %s",
```
Comment (liach) [inline src/main/resources/assets/modmenu/lang/zh_cn.json]: 用全角标点`：`

**Pull request merged**